fbchat/client.py

```diff
@@ -1747,7 +1758,7 @@ def markAsSeen(self):
         .. todo::
             Documenting this
         """
-        r = self._post(self.req_url.MARK_SEEN, {"seen_timestamp": 0})
+        r = self._post(self.req_url.MARK_SEEN, {"seen_timestamp": now()})
```

Comment:
Did you get `markAsSeen` to work?

Reply:
I don't really know what mark as seen is supposed to do :joy: but I saw in network tools that the sent timestamp is just actual timestamp, so I just changed it.

Reply:
Haha, okay, fair enough 👍